sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusClientBuilderTest.java

```diff
@@ -37,7 +37,7 @@
 
 class ServiceBusClientBuilderTest extends IntegrationTestBase {
     private static final String NAMESPACE_NAME = "dummyNamespaceName";
-    private static final String DEFAULT_DOMAIN_NAME = "servicebus.windows.net/";
+    private static final String DEFAULT_DOMAIN_NAME = TestUtils.getEndPoint().substring(1) + "/";
```

Comment:
It would be better to definte the domain name as a seperate entity instead of relying on the endpoint value.

Reply:
@ki1729  The endpoint suffix of Service bus is different on UsGov and Azure China cloud.  So it is more appropriate to relying on the endpoint value  . For details, please refer to Issue#[23646](https://github.com/Azure/azure-sdk-for-java/issues/23646)